gh-pr-context.py

```
@@ -88,16 +88,79 @@ def gh_api_jq(endpoint, jq_filter):
     return result.stdout.strip().replace("\r", ""), True
 
 
+def _setup_git_env():
+    """Resolve worktree gitdir when git rev-parse --git-dir fails.
+
+    Handles three cases:
+    1. Normal repo: git works, nothing to do.
+    2. Worktree with Windows absolute path (C:/...): convert to /mnt/c/...
+    3. Worktree with relative gitdir: resolve relative to worktree root.
+    """
+    out, rc = run_cmd(_git_cmd() + ["rev-parse", "--git-dir"])
```

Ruff (0.15.15)

[warning] Consider `[*_git_cmd(), "rev-parse", "--git-dir"]` instead of concatenation

Replace with `[*_git_cmd(), "rev-parse", "--git-dir"]`

(RUF005)